plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/address-form/address-form.tsx

```diff
@@ -163,6 +164,28 @@ const AddressForm = ( {
 					);
 				}
 
+				if ( field.type === 'select' ) {
+					if ( typeof field.options === 'undefined' ) {
+						return null;
+					}
+					
+					return (
+						<Combobox
+							key={ field.key }
+							{ ...fieldProps }
+							className="wc-block-components-select-input"
+							value={ values[ field.key ] }
```

Comment:
Shouldn't the class contain the key as well?

Reply:
Good idea, added in ddb4fb22fa0001bc51919cd886eb2207abf58576